index.html

```diff
@@ -9,6 +9,11 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
-  <h1 class="title">Hello World</h1> 
+  <header></header>
+  <main>
+    <h1 class="title">Front-End Tools</h1>
```

Comment:
for SEO reasons a good practice is to have a single h1 per page or an h1 per section in the same page. since we will add 2 sections in the future to wrap each section lets keep the h1 elements for now.